api/core/model_runtime/model_providers/fireworks/llm/llm.py

```diff
@@ -511,7 +511,7 @@ def _num_tokens_from_messages(
         model: str,
         messages: list[PromptMessage],
         tools: Optional[list[PromptMessageTool]] = None,
-        credentials: dict = None,
+        credentials: Optional[dict] = None,
```

Comment:
This parameter is unused, you can remove it.

Reply:
(ditto) Will remove the unused parameter in the follow-up PR, covering the method's signature and usages.